pymc/distributions/continuous.py

```diff
@@ -2628,15 +2628,18 @@ def dist(cls, nu=1, sigma=None, lam=None, sd=None, *args, **kwargs):
         lam, sigma = get_tau_sigma(lam, sigma)
         sigma = at.as_tensor_variable(sigma)
 
-        # mode = at.as_tensor_variable(0)
-        # median = at.as_tensor_variable(sigma)
-        # sd = at.as_tensor_variable(sigma)
-
         assert_negative_support(nu, "nu", "HalfStudentT")
         assert_negative_support(lam, "lam", "HalfStudentT")
         assert_negative_support(sigma, "sigma", "HalfStudentT")
 
         return super().dist([nu, sigma], *args, **kwargs)
+    
+    def get_moment(rv, size, nu, sigma):
+        median = at.as_tensor_variable(sigma)
+        sigma, _ = at.broadcast_arrays(sigma, nu)
+        if not rv_size_is_none(size):
+            median = at.full(size, median)
+        return median
```

Comment:
I think you want something like this:

```suggestion
        sigma, _ = at.broadcast_arrays(sigma, nu)
        if not rv_size_is_none(size):
            sigma = at.full(size, sigma)
        return sigma
```